tools/test.py

```diff
@@ -44,11 +44,14 @@
 import errno
 import copy
 import ast
+import resource
 
 from os.path import join, dirname, abspath, basename, isdir, exists
 from datetime import datetime
 from Queue import Queue, Empty
 
+resource.setrlimit(resource.RLIMIT_CORE, (0,0))
```

Comment:
I can take over if you want but what you have now is Good Enough, IMO.  Threading through the ulimit setting from test/abort/testcfg.py to subprocess.Popen() is quite a bit of work for not all that much gain.